src/react/reports/AutoScheduler/scheduler/schedule.ts

```diff
@@ -94,11 +89,7 @@ function planIssue(issue: LinkedIssue, workByTeams: TeamWorkIndex) {
 }
 
 // Try to find the first available time  the work could be scheduled for this team.
-function scheduleIssue(
-  linkedIssue: LinkedIssue,
-  firstDayWorkCouldStartOn: number,
-  teamWorkIndex: TeamWorkIndex
-) {
+function scheduleIssue(linkedIssue: LinkedIssue, firstDayWorkCouldStartOn: number, teamWorkIndex: TeamWorkIndex) {
```

Comment:
is there a way to allow for params to be broken up on multiple lines?  When there are a lot of params + types, it can be a bit hard to read ..